.changes/next-release/Feature-015cabbc-aec3-4b75-92e9-10489e8ee7be.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Add new IDE type SM VSCode"
```

Comment:
The changelog is for customers, it's our lowest-friction form of marketing. So changelog items should be meaningful to customers. Is this description going to mean anything to customers that read it?

Reply:
Understood, sent you a slack, let me know how you like me to proceed.